acceptance/tests/ensure_facter_3_and_facter_4_output_matches.rb

```diff
@@ -1,57 +1,52 @@
 test_name 'Ensure Facter 3 and Facter 4 outputs match' do
   require 'puppet/acceptance/common_utils'
-  require 'puppet/acceptance/fact_dif'
```

Comment:
`puppet/acceptance/fact_dif.rb` can be removed, this was the only place that we used it.